airflow/bin/cli.py

```diff
@@ -410,18 +410,28 @@ def webserver(args):
         app.run(debug=True, port=args.port, host=args.hostname)
     else:
         pid, stdout, stderr, log_file = setup_locations("webserver", pid=args.pid)
+
+        ssl_certfile=configuration.get('webserver', 'ssl_certfile')
```

Comment:
Have you tried this with the ssl_certfile and ssl_keyfile fields commented out in configuration.py? I'm fairly certain that this will throw an Airflow exception:

```
    def get(self, section, key, **kwargs):
        section = str(section).lower()
        key = str(key).lower()

        d = self.defaults

        # first check environment variables
        option = self._get_env_var_option(section, key)
        if option:
            return option

        # ...then the config file
        if self.has_option(section, key):
            return expand_env_var(
                ConfigParser.get(self, section, key, **kwargs))

        # ...then commands
        option = self._get_cmd_option(section, key)
        if option:
            return option

        # ...then the defaults
        if section in d and key in d[section]:
            return expand_env_var(d[section][key])

        else:
            logging.warn("section/key [{section}/{key}] not found "
                         "in config".format(**locals()))

            raise AirflowConfigException(
                "section/key [{section}/{key}] not found "
                "in config".format(**locals()))
```

Since ssl_certfile and ssl_keyfile will not be defined in the config, defaults, etc.